docs/introduction/how-to-work-with-money.md

```diff
@@ -0,0 +1,388 @@
+{{ money|moneyFormat(null, '.', ',') }}  {# renders "1,234.5670" #}
+```
+
+### price
+
+Filter `price` formats the amount of money in a localized manner.
+It uses basic frontend currency on the current domain and the current locale (language) to format it.
+
+For example, one thousand Czech crowns would be rendered as *"CZK1,000.00"* on an English domain and as *"1 000,00 Kč"* on a Czech one.
+
+All `price*` Twig filters use this format.
+They usually differ only in the currency and locale they use.
+
+### priceText
+
+Filter `priceText` formats the amount of money in a localized manner, similarly to the `price` filter.
+The only difference is that it outputs the text *"Free"* when zero amount of money is provided.
+
+### priceTextWithCurrencyByCurrencyIdAndLocale
+
+Filter `priceTextWithCurrencyByCurrencyIdAndLocale` can be used to format the amount of money in any currency and locale.
+It outputs the text *"Free"* when zero amount of money is provided.
+
+The *currency ID* (`int`) and the *locale* (`string`) must be provided as parameters.
+
+### priceWithCurrency
+
+Filter `priceWithCurrency` can be used to format the amount of money in any currency.
+It uses the current locale (the locale of current domain or administration) to format it.
+
+The *currency* (`Currency` entity) must be provided as a parameter.
+
+### priceWithCurrencyAdmin
+
+It works similarly to `priceWithCurrency`, but it uses the default administration currency.
+
+It has no parameters.
+
+### priceWithCurrencyByCurrencyId
+
+It works similarly to `priceWithCurrency`, but it uses currency ID instead of the `Currency` entity.
+
+The *currency ID* (`int`) must be provided as a parameter.
+
+### priceWithCurrencyByDomainId
+
+It works similarly to `priceWithCurrency`, but it uses the basic frontend currency of the provided domain.
+
+The *domain ID* (`int`) must be provided as a parameter.
+
+## Money in Javascript
+
+`Money` implements the `JsonSerializable` interface, which means it can be serialized into JSON format with `json_encode` into an object with the following structure:
+
+```json
+{
+  "amount": "1234.5670"
+}
+```
+
+This format can be readily used in Javascript when JSON is rendered in Twig:
+
+```twig
+{# the "money" variable contains Money::create('1234.5670') #}
+
+<div id="my-money" data-money="{{ money|json_encode }}"/>
+```
+
+```js
+(function ($) {
+
+    $(document).ready(function () {
+        var money = $('#my-money').data('money');
+
+        // Displays "1234.5670"
+        alert(money.amount);
+    });
+
+})(jQuery);
+```
+
+See the native Javascript method [`.toFixed()`](https://www.w3schools.com/jsref/jsref_tofixed.asp) and the function [`parseFloat()`](https://www.w3schools.com/jsref/jsref_parsefloat.asp) to see how to convert between decimal strings and numbers.
+
+## Money in Doctrine
+
+In Shopsys Framework, there is a custom Doctrine type `money` which can be used in similar fashion as `decimal` type.
```

Comment:
```suggestion
We use a custom Doctrine type `money` which can be used in similar fashion as `decimal` type.
```

Reply:
There is no "we" or "us" in this article and I'd like to keep it consistent.